addons/web/models/models.py

```diff
@@ -780,7 +780,7 @@ def _get_asset_style_b64(self):
         company_styles = template_style._render({
             'company_ids': company_ids,
         })
-        return base64.b64encode((company_styles))
+        return base64.b64encode(company_styles.encode('utf-8'))
```

Comment:
utf-8 is the default
```suggestion
        return base64.b64encode(company_styles.encode())
```